src/math/trigonometry.js

```diff
@@ -42,14 +42,14 @@ p5.prototype._angleMode = constants.RADIANS;
  *
  * <div>
  * <code>
- * let a = PI;
+ * let a = PI-QUARTER_PI;
```

Comment:
Please add space around the operator for readability, as in `PI - QUARTER_PI`

Reply:
Thanks, I made the desired changes, and also changed `a` to `PI + QUARTER_PI` so the output would exactly match the statement we have currently in the `describe` function.

Reply:
Looks good!